src/guessTheNumber.js

```diff
@@ -1,3 +1,19 @@
 export default function guessTheNumber() {
-  // your code here
+  var randomNum = Math.floor(Math.random() * (100)) + 1
```

Comment:
There has to be a better way to do this. have you looked into the other Math functions?